cms/apps/media/admin.py

```diff
@@ -2,11 +2,17 @@
 from functools import partial
 
 import requests
+from cms import permalinks
+from cms.apps.media.forms import FileForm, ImageChangeForm
+from cms.apps.media.models import File, Label, Video
```

Comment:
These imports properly belonged in their previous location because they are package-local.